client/option.go

```diff
@@ -0,0 +1,58 @@
+package client
+
+import (
+	"time"
+)
+
+// Option is a function that can be passed to NewAPIClient to configure it.
```

Comment:
feature request: option to override the underlying http.Client object.

Reply:
Will be implemented in the next change set.